test/functional/kaui/accounts_controller_test.rb

```diff
@@ -129,4 +149,40 @@ class Kaui::AccountsControllerTest < Kaui::FunctionalTestHelper
     post :pay_all_invoices, :account_id => @account.account_id, :is_external_payment => true
     assert_response 302
   end
+
+  test 'should trigger invoice' do
+    parameters = {
+      :account_id => @account2.account_id,
+      :dry_run => '0'
+    }
+
+    post :trigger_invoice, parameters
+    assert_equal 'Nothing to generate for target date today', flash[:notice]
+    assert_redirected_to account_path(@account2.account_id)
+
+    today_next_month = (Date.today >> 1).to_s
```

Comment:
To make the tests more robust, I would suggest to look at the clock in Kill Bill instead of relying on `Date.today` (just in case one test modifies the clock and doesn't reset it).

Similar remarks below.